testing/e2e/app/routes/ComboBox/route.tsx

```diff
@@ -1,24 +1,51 @@
-import { ComboBox } from '@itwin/itwinui-react';
+import { Button, ComboBox } from '@itwin/itwinui-react';
 import { useSearchParams } from '@remix-run/react';
+import React from 'react';
 
 export default function ComboBoxTest() {
   const [searchParams] = useSearchParams();
 
   const virtualization = searchParams.get('virtualization') === 'true';
+  const multiple = searchParams.get('multiple') === 'true';
+
+  const options = [
+    { label: 'Item 0', value: 0 },
+    { label: 'Item 1', value: 1, subLabel: 'sub label' },
+    { label: 'Item 2', value: 2 },
+    { label: 'Item 3', value: 3 },
+    { label: 'Item 4', value: 4 },
+    { label: 'Item 10', value: 10 },
+    { label: 'Item 11', value: 11 },
+  ];
+
+  const initialValueSearchParam = searchParams.get('initialValue') as
+    | ('all' & string & {})
+    | null;
+  const initialValue =
+    initialValueSearchParam != null
+      ? initialValueSearchParam === 'all'
+        ? options.map((option) => option.value)
+        : (JSON.parse(initialValueSearchParam) as number | number[])
+      : undefined;
+
+  const [value, setValue] = React.useState(initialValue);
 
   return (
-    <ComboBox
-      options={[
-        { label: 'Item 0', value: 0 },
-        { label: 'Item 1', value: 1, subLabel: 'sub label' },
-        { label: 'Item 2', value: 2 },
-        { label: 'Item 3', value: 3 },
-        { label: 'Item 4', value: 4 },
-        { label: 'Item 10', value: 10 },
-        { label: 'Item 11', value: 11 },
-      ]}
-      id='test-component'
-      enableVirtualization={virtualization}
-    />
+    <div data-testid='container'>
+      <Button
+        data-testid='change-value-to-first-option-button'
+        onClick={() => setValue(multiple ? [0] : 0)}
+      >
+        Change value to just the first option
+      </Button>
```

Comment:
all the code for new tests should be placed in a separate component and conditionally rendered by the default component

in other words: adding new tests should not affect existing tests where possible. right now, you need to change the existing tests to account for the extra button